grails-app/domain/org/pih/warehouse/requisition/Requisition.groovy

```diff
@@ -46,6 +46,7 @@ class Requisition implements Comparable<Requisition>, Serializable {
     Date dateDelivered
     Date dateIssued
     Date dateReceived
+    Date desiredDeliveryDate
```

Comment:
Looks like our convention is `date<Something>`, but I don't know how this should look like for this one. @jmiranda do you have any preference or suggestion?

Reply:
@awalkowiak @alannadolny I will have an answer for you today.

Reply:
In the future, let's flag all tickets that require data model changes during the planning poker activity. All data model changes (no matter how small) must go through a design process. And this should almost always be done before we start working on the feature. 

cc @awalkowiak @EWaterman

Reply:
I think dateDeliveryRequested makes the most sense. 

Besides the naming, the reason I mentioned "design" was because I would have implemented this a little more generically at the data model and API level. The requested delivery date should also be available at the requisition line item level, even if the UI of the current feature does not support it.